cmd/cigarToBed/cigarToBed_test.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"github.com/vertgenlab/gonomics/bed"
+	"github.com/vertgenlab/gonomics/common"
+	"github.com/vertgenlab/gonomics/fileio"
+	"os"
+	"testing"
+)
+
+var CigarToBedTests = []struct {
+	inputFileOne_Name   string
+	inputFileTwo_Name   string
+	outFa               string
+	FirstPos_InsBed     int
+	FirstPos_DelBed     int
+	Chrom               string
+	outIns_bed_expected string
+	outDel_bed_expected string
+}{
+	{"testdata/sethvsraven/seth.fa", "testdata/sethvsraven/raven.fa", "", 1, 1, "chr1", "testdata/sethvsraven/affineGap_sethvsraven_ins.bed", "testdata/sethvsraven/affineGap_sethvsraven_del.bed"},
+	{"testdata/firstTest/testRegion10kb_PanTro6.fa", "testdata/firstTest/testRegion10kb_hg38.fa", "", 119320000, 116703287, "chr1", "testdata/firstTest/affineGap_PanTro6vshg38_ins.bed", "testdata/firstTest/affineGap_PanTro6vshg38_del.bed"},
+}
+
+func TestCigarToBed(t *testing.T) {
+	for _, v := range CigarToBedTests {
+		inputFileOne := fileio.EasyOpen(v.inputFileOne_Name)
+		inputFileTwo := fileio.EasyOpen(v.inputFileTwo_Name)
+		GlobalAlignment_CigarToBed(inputFileOne, inputFileTwo, v.outFa, "ins_tmp.bed", "del_tmp.bed", v.FirstPos_InsBed, v.FirstPos_DelBed, v.Chrom)
+
+		records_ins := bed.Read("ins_tmp.bed")
+		expected_ins := bed.Read(v.outIns_bed_expected)
+		if !bed.AllAreEqual(records_ins, expected_ins) {
```

Comment:
I've recently learned that bed.AllAreEqual only checks the first three columns of the bed for matching. It doesnt compare past there. Just a heads up, as it wont be comparing the name column which I believe you are using in your system here (ie column 4).

Reply:
@RavenYL I recommend actually making this change a priority or your test file might not be functioning. You can run fileio.AreEqual(ins_tmp.bed, v.outIns_bed_expected) I believe to do the testing until the bed.AllAreEqual() is fixed (really the change will be in bed.Equal() which bed.AllAreEqual() depends on).

Reply:
I just switched to fileio.AreEqual and commented on the bed.AllAreEqual situation in my new PR. Thank you!